.github/workflows/continuous-integration.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: "actions/cache@v1"
         with:
           path: "~/.composer/cache"
-          key: "composer-${{ hashFiles('composer.json') }}"
+          key: "composer-${{ hashFiles('composer.lock') }}"
```

Comment:
:thinking: I will copy that in #3978 I guess. This commit should probably go since `composer update` is still in use here.